src/index.html

```diff
@@ -6,9 +6,24 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@800&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <a href="index.html" class="a_img"><img src="images/logo.png" alt="The logo of Moyo"></a>
```

Comment:
a_img in not correct name, try use BEM for naming, for instance:
```suggestion
      <a href="index.html" class="head__logo"><img src="images/logo.png" alt="The logo of Moyo"></a>
```